packages/roosterjs-editor-api/lib/cursor/replaceTextBeforeCursorWithNode.ts

```diff
@@ -86,13 +86,19 @@ function validateAndGetRangeForTextBeforeCursor(
     return textIndex == -1 ? range : null;
 }
 
-// Replace text before cursor with a node
-// exactMatch is to match exactly, i.e.
-// exactMatch = true case, in Mentions, you type @nick, and then pick a suggestion from picker, we need to replace exactly everything before cursor (@nick)
-// with the suggestion from Mentions picker
-// exactMatch = false case, in auto linkification, users could type "www.bing.com,<space>". The auto link will kick in on space
-// at the moment, what is before cursor is "www.bing.com,", however, only "www.bing.com" makes the link. by setting exactMatch = false, it does not match
-// right from the end, but can scan through till first same char is seen.
+/**
+ * Replace text before cursor with a node
+ * @param editor The editor instance
+ * @param text The text for matching. We will try to match the text with the text before cursor
+ * @param node The node to replace the text with
+ * @param exactMatch exactMatch is to match exactly, i.e.
+ * exactMatch = true case, in Mentions, you type @nick, and then pick a suggestion from picker, we need to replace exactly everything before cursor (@nick)
+ * with the suggestion from Mentions picker
+ * exactMatch = false case, in auto linkification, users could type "www.bing.com,<space>". The auto link will kick in on space
+ * at the moment, what is before cursor is "www.bing.com,", however, only "www.bing.com" makes the link. by setting exactMatch = false, it does not match
+ * right from the end, but can scan through till first same char is seen.
+ * @param cursorData
```

Comment:
Don't use mention as example since roosterjs doesn't have mention. If there isn't a good example, just describe it without an example.

Reply:
Removed mention as example